src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/Models/ContainerService.cs

```diff
@@ -24,26 +24,32 @@ public partial class ContainerService : Resource
-        /// Agent</param>
-        public ContainerService(string location, ContainerServiceMasterProfile masterProfile, System.Collections.Generic.IList<ContainerServiceAgentPoolProfile> agentPoolProfiles, ContainerServiceLinuxProfile linuxProfile, string id = default(string), string name = default(string), string type = default(string), System.Collections.Generic.IDictionary<string, string> tags = default(System.Collections.Generic.IDictionary<string, string>), string provisioningState = default(string), ContainerServiceOrchestratorProfile orchestratorProfile = default(ContainerServiceOrchestratorProfile), ContainerServiceWindowsProfile windowsProfile = default(ContainerServiceWindowsProfile), ContainerServiceDiagnosticsProfile diagnosticsProfile = default(ContainerServiceDiagnosticsProfile))
+        /// <param name="location">Resource location.</param>
+        /// <param name="masterProfile">Properties of master agents.</param>
+        /// <param name="agentPoolProfiles">Properties of the agent
+        /// pool.</param>
+        /// <param name="linuxProfile">Properties of Linux VMs.</param>
+        /// <param name="id">Resource ID.</param>
+        /// <param name="name">Resource name.</param>
+        /// <param name="type">Resource type.</param>
+        /// <param name="tags">Resource tags.</param>
+        /// <param name="provisioningState">The current deployment or
+        /// provisioning state, which only appears in the response.</param>
+        /// <param name="orchestratorProfile">Properties of the
+        /// orchestrator.</param>
+        /// <param name="customProfile">Properties for custom clusters.</param>
+        /// <param name="servicePrincipalProfile">Properties for cluster
+        /// service principals.</param>
+        /// <param name="windowsProfile">Properties of Windows VMs.</param>
+        /// <param name="diagnosticsProfile">Properties of the diagnostic
+        /// agent.</param>
+        public ContainerService(string location, ContainerServiceMasterProfile masterProfile, System.Collections.Generic.IList<ContainerServiceAgentPoolProfile> agentPoolProfiles, ContainerServiceLinuxProfile linuxProfile, string id = default(string), string name = default(string), string type = default(string), System.Collections.Generic.IDictionary<string, string> tags = default(System.Collections.Generic.IDictionary<string, string>), string provisioningState = default(string), ContainerServiceOrchestratorProfile orchestratorProfile = default(ContainerServiceOrchestratorProfile), ContainerServiceCustomProfile customProfile = default(ContainerServiceCustomProfile), ContainerServiceServicePrincipalProfile servicePrincipalProfile = default(ContainerServiceServicePrincipalProfile), ContainerServiceWindowsProfile windowsProfile = default(ContainerServiceWindowsProfile), ContainerServiceDiagnosticsProfile diagnosticsProfile = default(ContainerServiceDiagnosticsProfile))
```

Comment:
@brendandburns this is a breaking change - parameters have been added to the constructor and the order of some of the last parameters have changed